release-handbook/doc_roles_handbook.md

```diff
@@ -0,0 +1,68 @@
+# Zowe documentation squad roles
+
+This document covers the responsibilities, time commitments, and basic processes and guidance for different roles within the Zowe Docs team. 
+
+## Zowe docs roles
+
+- Zowe Docs Lead
+- Zowe Docs Scrum Master
+- Zowe Docs Lead Shadows/Co-lead
+- Zowe TSC Rep for Doc Squad
+- Zowe Release Lead
+
+## Docs Lead Responsibilities
+
+The Docs Lead is responsible for coordinating documentation updates for new Zowe releases, managing the Zowe docs-site repo, and leading the squad to define documentation roadmap and strategy.
+
+Responsibilities include:
+
+Task | Where | Time requirements | Resources, comments 
+---| ---| ---| ---
+Monitor Zowe docs-site issues and assign owners | Zowe docs-site GitHub repo | 1-2 hours a week | Looking to automate the process
+Monitor Zowe doc-site Pull Requests and assign reviewers | Zowe docs-site GitHub repo | 1 hour a week | 
+Identify and track new Zowe documentation requests and feedback in other channels like Slack, provide response, and convert into actionable issues | Slack, meetings where invited, emails | Depends, usually 2-3 hours a week | As a squad lead, you'll be the go-to-person whenever people have questions, comments, or ideas about documentation so might get involved in different discussions.
```

Comment:
Collaboration needed here. Members might have different channels to collect feedback and receive requests, and can work together on the response or actions. And the communicator could be a volunteer or the one who receives the request.